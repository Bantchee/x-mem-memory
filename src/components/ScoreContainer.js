import CurrentScore from "./CurrentScore";
import BestScore from "./BestScore";

const ScoreContainer = (props) => {
    return (
        <div
            className="flex text-2xl gap-16 justify-center"
        >
            <CurrentScore
                score={props.currentScore}
            />
            <BestScore 
                score={props.bestScore}
            />
        </div>
    )
}

export default ScoreContainer;