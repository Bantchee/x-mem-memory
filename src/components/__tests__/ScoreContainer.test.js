import ScoreContainer from "../ScoreContainer";
import { render } from "@testing-library/react";

describe("ScoreContainer Component", () => {
    test("render component", () => {
        const { container } = render(<ScoreContainer/>);
        expect(container).toMatchSnapshot();
    });
});