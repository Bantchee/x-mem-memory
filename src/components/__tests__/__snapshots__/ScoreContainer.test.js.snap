// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScoreContainer Component render component 1`] = `
<div>
  <div
    class="flex text-2xl gap-16 justify-center"
  >
    <div
      class="pt-1 pb-1 border-4 w-64 flex justify-center rounded-lg border-yellow-400"
    >
      <span
        class="text-red-500 m-auto"
      >
        Current Score: 
      </span>
    </div>
    <div
      class="pt-1 pb-1 border-4 w-64 flex justify-center rounded-lg border-yellow-400"
    >
      <span
        class="text-red-500 m-auto"
      >
        Best Score: 
      </span>
    </div>
  </div>
</div>
`;
